lib/Runtime/Library/JavascriptLibrary.h

```diff
@@ -605,7 +605,6 @@ namespace Js
         JavascriptFunction* GetStringConstructor() const {return stringConstructor; }
         JavascriptFunction* GetArrayBufferConstructor() const {return arrayBufferConstructor; }
         JavascriptFunction* GetErrorConstructor() const { return errorConstructor; }
-        JavascriptFunction* GetTypedArrayConstructor() const { return typedArrayConstructor; }
```

Comment:
Not this line. This is duplicated from JavascriptLibraryBase.h
Yes for the rest.  Noticed it when I was looking at the profile the other day.  This reduce the binary size by ~50K in x86 and ~80K in amd64 (release build), because all these templated DeferTypeHandler code for the non-prototype variant is gone.